.devcontainer/Dockerfile

```
@@ -56,3 +56,13 @@ RUN git clone https://github.com/pyenv/pyenv.git ${PYENV_ROOT} \
     && python -m pip install --upgrade pip \
     && pip install black flake8 mypy numpy openfermion pybind11-stubgen scipy wheel \
     && chmod -R 777 /.pyenv
+
+# Install cereal
+RUN git clone https://github.com/USCiLab/cereal.git -b v1.3.0 /tmp/cereal \
```

`--depth 1`もつけておくと、Clone時のダウンロードサイズが小さくなってビルド時間の短縮にもなりそうです

```suggestion
RUN git clone https://github.com/USCiLab/cereal.git -b v1.3.0 --depth 1 /tmp/cereal \
```